translations/ES_eu (international Spanish really)

```diff
@@ -0,0 +1,128 @@
+        - question: "¿Bitcoin Cash es diferente de 'Bitcoin'?"
+          answer:
+            - |
+              Sí. Bitcoin Cash es la continuación del proyecto Bitcoin como dinero digital de usuario a usuario. Es una bifurcación de la blockchain de Bitcoin, con reglas de consenso actualizadas que le permiten crecer y escalar.
+        - question: "¿Si poseo Bitcoin, poseo también Bitcoin Cash automáticamente?"
+          answer:
+            - |
+              Todo aquel que poseía Bitcoin en el momento de la creación de Bitcoin Cash, pasó a tener esa misma cantidad en Bitcoin Cash. Esto significa que todo el que poseía Bitcoin en el momento del bloque 478558 (el 1 de agosto de 2017, alrededor de las 13:16 UTC) obtuvo una cantidad de Bitcoin Cash equivalente a la que tenía de Bitcoin en ese momento. Si sus Bitcoins los guarda algún tercero, como por ejemplo una casa de cambio, tendrá que tratar el asunto de sus Bitcoin Cash con ese tercero.
+            - |
+              Todas las transacciones de Bitcoin y Bitcoin Cash posteriores a la bifurcación del 1 de agosto son totalmente independientes las unas de las otras. Por este motivo, toda cantidad de Bitcoin adquirida tras la bifurcación no incluye nada de Bitcoin Cash, y viceversa.
+        - question: "¿Cómo se trata el asunto de la repetición de las transacciones entre la cadena antigua y la nueva?"
+          answer:
+            - |
+              Las transacciones de Bitcoin Cash transactions usan un algoritmo de hashing de firmas y vienen indicadas por la etiqueta SIGHASH_FORKID. Estas firmas no son válidas en la red de Bitcoin antigua. Esto evita que las transacciones de Bitcoin Cash se reproduzcan en la cadena de Bitcoin y viceversa.
+        - question: "¿Por qué fue necesaria una bifurcación (<i>fork</i>) para crear Bitcoin Cash?"
+          answer:
+            - |
+              El código de Bitcoin establecía un límite máximo de datos de 1MB por bloque, el equivalente a 3 transacciones por segundo. Aunque aumentar este límite era algo sencillo, la comunidad no pudo alcanzar un consenso a pesar de debatirlo durante años.
+        - question: "¿El límite de 1MB por bloque causaba problemas a Bitcoin?"
+          answer:
+            - |
+              Sí. En 2017 la capacidad alcanzó el "umbral invisible". Las comisiones aumentaron drásticamente y las transacciones de Bitcoin se volvieron poco fiables, dándose el caso de que algunos usuarios no conseguián que sus transacciones se confirmasen incluso tras días de espera.
+            - |
+              El precio de mercado de Bitcoin ha aumentado, pero su crecimiento y utilidad como moneda se han estancado. Muchos usuarios, comerciantes, empresas e incluso inversores abandonaron Bitcoin en busca de alternativas, por lo que el dominio de Bitcoin descendío del 95% hasta alrededor del 40%.
```

Comment:
`descendío` -> `descendió`

"a números tan bajos como el 40%" feels more accurate to me than "hasta alrededor del 40%" since that number has increased now.

Reply:
I agree

Reply:
Wrote "una cifra tan baja como el 40%", do you like?